core/common/src/TerrainSettings.ts

```diff
@@ -25,6 +25,8 @@ export type TerrainProviderName = string;
 export enum CesiumTerrainAssetId {
```

Comment:
wonder if we want to internally maintain a list of all publicly curated cesium content
https://developer.bentley.com/apis/cesium-curated-content/operations/list-content/